### ISSUE TITLE
Automatic build of Workbench [CARROT-175]

### ISSUE THREAD

**Issue by dweiss** [carrot2/carrot2#381]
Workbench app + Carrot2 3.0 should be build outside of Eclipse PDE. It would be great, if products (for many platforms) could be created the same way.

It can be Ant build using Eclipse headless build feature.

---
Issue: CARROT-175 (migrated from JIRA), created by Urszula Krukar, resolved Apr 04 2008

**Comment by dweiss** [comment]
Comment by Urszula Krukar (migrated from JIRA)

Hurra!

I managed to build workbench product in headless mode!!

Problem is, it cannot be done the way I did it with out current folder structure :( (it requires plugins in one folder, features in the other, like normal eclipse folder structure). I will see if I can override it.

**Comment by dweiss** [comment]
Comment by Stanisław Osiński (@stanislawosinski) (migrated from JIRA)

Reassigning for me to add Workbench building to Bamboo.

**Comment by dweiss** [comment]
Comment by Urszula Krukar (migrated from JIRA)

I managed to test workbench product in headless mode!! :D

Result of a test is a XML file (formatter: org.apache.tools.ant.taskdefs.optional.junit.XMLJUnitResultFormatter). Will this be a problem?

So far only JUnit 3 is working (not sure why exactly), I also had to make all packages from org.carrot2.workbench.core plugin visible at runtime. Will try to work around that soon.
